lib/gui/os/window-progress.js

```diff
@@ -18,6 +18,40 @@
 
 const electron = require('electron');
 const utils = require('../../shared/utils');
+const progressStatus = require('../modules/progress-status');
+
+/**
+ * @summary The title of the main window upon program launch
+ * @type {String}
+ * @private
+ * @constant
+ */
+const initialTitle = document.title;
+
+/**
+ * @summary Make the full window status title
+ * @private
+ *
+ * @param {String} baseTitle - base window title
+ * @param {Object} currentFlashState - flash state object
+ *
+ * @returns {String}
+ *
+ * @example
+ * const title = makeTitle({
```

Comment:
I now noticed we can simplify the function to not need that argument, so let me make that change.